Payload/bleemsync/etc/bleemsync/FUNC/0050_bleemsync.funcs

```diff
@@ -290,11 +208,10 @@ execute_bleemsync_func(){
   start=$(date +%s%N)
   #Cleanup before starting main user applications
   rm -rf "/tmp/systemd"*
-  rm -rf "/tmp/diag"
+  rm -rf "/tmp/diag/"*
```

Comment:
This will prevent deleting `/tmp/diag/.tmp`, which contains debug logs generated by the bootloader and redirected by `usb_watch`.